.travis.yml

```diff
@@ -12,12 +12,8 @@ before_install:
   - npm config set ca ""
 install:
   - curl -L https://rawgit.com/marionettejs/travis-ci-setup/master/ci.sh | bash
+  - npm install travis-ci
 after_success:
   - if [[ $MAINRUN == true ]]; then npm run-script coveralls; fi
-before_deploy:
-  - npm install travis-ci
-deploy:
```

Comment:
It was my mistake adding deploy option, which cost us wrong release...
[Here](https://docs.travis-ci.com/user/customizing-the-build#Deploying-your-Code) some doc about deploy option, I think it will explain better then I..